pkg/osquery/table/gdrive_sync.go

```diff
@@ -0,0 +1,61 @@
+func GDrivePlugin(client *osquery.ExtensionManagerClient) *table.Plugin {
+	paths := queryDbPath...
+	database, _ := sql.Open("sqlite3", path)
+	g := &gdrive{
+		db: database,
+	}
+
+	columns := []table.ColumnDefinition{
+		table.TextColumn("user_email"),
+		table.TextColumn("local_sync_root_path"),
+	}
+	return table.NewPlugin("kolide_gdrive_sync_config", columns, generate)
```

Comment:
should be `g.generate`, since you're calling the generate function on the `g` struct`